shared/src/PayloadConvention.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Shared
+{
+    /// <summary>
+    /// The payload convention class.
+    /// </summary>
+    /// <remarks>The payload convention is used to define a specific serializer as well as a specific content encoding. For example, IoT has a <see href="https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-convention">convention</see> that is designed to make it easier to get started with products that use specific conventions by default.</remarks>
```

Comment:
can we break this line up. it is a bit long

Reply:
would this be better?
For example, IoT has a <see href="https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-convention">convention</see> that is designed
to make it easier to get started with convention-based products.

Reply:
I don't mean to reword it, just to line break at a reasonable length

Reply:
ah, got it!